update_changelog.sh

```diff
@@ -19,13 +19,13 @@ features_for() {
 
 gen_changelog() {
   {
-  echo "# Changelog"
+  echo "# Latest release notes"
```

Comment:
As this includes all versions right now (something that might be worth rethinking), probably don't say _Latest_
```suggestion
  echo "# Release notes"
```

Reply:
Done!